### ISSUE TITLE
`new_chat_members` doesn't work in channel

### ISSUE THREAD

**Issue by tourcoder** [yagop/node-telegram-bot-api#904]
> Bug Report

I have read:

* [Usage information](https://github.com/yagop/node-telegram-bot-api/tree/master/doc/usage.md)
* [Help information](https://github.com/yagop/node-telegram-bot-api/tree/master/doc/help.md)

I am using the latest version of the library.

### Expected Behavior

When a user subscribes to my channel, the bot can get the information of that user, like it does in a Telegram Group.

```
bot.on('new_chat_members', async (msg) => {
    console.log(msg);
});
```

It will print the new subscriber's information.

### Actual Behavior

The bot can't get the new subscriber's information, there's no response, even

```
bot.on('message', async (msg) => {
    console.log(msg);
});
```

got nothing.

### What I tried

I tried

```
bot.on("channel_post", async (msg) => {
    console.log(msg);
});
```

It only can get the post response.

Any ideas?

**Comment by kamikazechaser** [comment]
Yes, with `td-cli` it should be possible to keep track, but it isn't straight forward.

**Comment by tourcoder** [comment]
I see, thank you.
